jazz_create-serverless-service/index.js

```diff
@@ -73,7 +75,26 @@ module.exports.handler = (event, context, cb) => {
             })
             .catch(function (err) {
                 logger.error('Error while creating a service : ' + JSON.stringify(err));
-                return cb(JSON.stringify(errorHandler.throwInternalServerError(err.message)));
+                if(err.jenkins_api_failure) {
+                        logger.error('Jenkins API Failed!!!');
```

Comment:
We already logged the error above (line 77) so this is redundant.

Reply:
Removed the log.